tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -319,7 +330,21 @@ private void SetAgentUriAndTransport(Uri uri)
         {
             if (uri.OriginalString.StartsWith(UnixDomainSocketPrefix, StringComparison.OrdinalIgnoreCase))
             {
-                SetUdsAsTraceTransportAndCheckFile(uri.PathAndQuery);
+                TracesTransport = TracesTransportType.UnixDomainSocket;
+                TracesUnixDomainSocketPath = uri.PathAndQuery;
+
+                var absoluteUri = uri.AbsoluteUri.Replace(UnixDomainSocketPrefix, string.Empty);
```

Comment:
You used `OriginalString` previously, and `AbsoluteUri` here. I think we should stick with `OriginalString`?

Reply:
I've used AbsoluteUri as it seemed to be the best option. Because:
-  if I use `PathAndQuery`, then I get an absolute path missing a part
- AbsoluteUri is adding an extra `/` for windows absoluteURI.

That said I should use IsPathRooted instead of just checking if the path starts with a `/`